client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -46,8 +47,9 @@ const USPS: React.FC< Props > = ( {
 } ) => {
 	const translate = useTranslate();
 
-	const isAnnualPeriod = billingPeriod === IntervalLength.ANNUALLY;
+	const isAnnualPlan = useSelector( isAnnualPlanOrUpgradeableAnnualPeriod );
 
+	const isAnnualPeriod = billingPeriod === IntervalLength.ANNUALLY;
 	const planDisplayCost = useSelector( ( state ) =>
 		getProductDisplayCost( state, isAnnualPeriod ? PLAN_BUSINESS : PLAN_BUSINESS_MONTHLY )
```

Comment:
```suggestion
	const isAnnualPlan = useSelector( isAnnualPlanOrUpgradeableAnnualPeriod );

	const planDisplayCost = useSelector( ( state ) =>
		getProductDisplayCost( state, isAnnualPlan ? PLAN_BUSINESS : PLAN_BUSINESS_MONTHLY )
```

Reply:
We would be changing previous logic in that case because `isAnnualPeriod` contains the billing period selected when purchasing a plugin while the new `isAnnualPlan` contains logic related to the plan selected by the user. 
In other words, there is a scenario where `isAnnualPeriod` could be `true` while `isAnnualPlan` could be `false`. For example, if the user has a _Business Monthly_ plan but selects _Annual_ billing period when purchasing the plugin.

Reply:
That's a good point! My thinking was that the new selector kind of excludes this scenario, because we return either `isAnnualPeriod` or `isAnnualPlan` based on the value of `shouldUpgrade`.

As the only place it's then used is also guarded with `shouldUpgrade`, I'm good with going forward with either version! 🎉 🚢

Reply:
That is totally correct! Although, in this scenario, I would prefer to leave the current version for `planDisplayCost` because it reflects a more specific condition. So if you are happy with the current version, I will go ahead and merge the issue.

Thanks for your comments!